bp_common/test/src/bootrom/Makefile

```diff
@@ -1,8 +1,26 @@
 
-RISCV_GCC = $(CROSS_COMPILE)gcc -fPIC -march=rv64im -mabi=lp64 -mcmodel=medany -static -nostdlib -nostartfiles
```

Comment:
was fPIC causing problems?

Reply:
Yup, it was creating position independent code and because of that the GOT was being generated. I got rid of this option and put everything in one section for a flat compilation.

Reply:
Makes sense!